src/routes/components/components.json

```
@@ -853,6 +853,16 @@
 		"title": "svelte-material-ui",
 		"url": "https://github.com/hperrin/svelte-material-ui"
 	},
+	{
+		"addedOn": "2020-09-29T14:39:13Z",
+		"category": "Design System",
+		"description": "A Svelte.js based UnoCSS UI library that allows you to make websites",
+		"npm": "@ikun-ui/core",
+		"stars": 650,
+		"tags": ["components and libraries", "component sets"],
+		"title": "ikun-ui",
+		"url": "https://github.com/ikun-svelte/ikun-ui"
```

Since very recently (few minutes ago), this field should be used to denote a homepage different from the repository.
And the URL of the repository should be put in a field named `repository`
```suggestion
		"repository": "https://github.com/ikun-svelte/ikun-ui"
```